src/resolvers.ts

```diff
@@ -17,7 +17,7 @@ const resolvers = {
             return queryResults.data
         },
         facilities: async (_parent: unknown, args: { filters: gqlType.FacilitySearchFilters })
-        : Promise<gqlType.Facility[]> => {
+        : Promise<gqlType.SearchResultsFacilities> => {
```

Comment:
Since we're changing the object returned to the client side, it's going tovbreak the api requests. 
To solve this we should version this endpoint as v2